Shoko.Server/Scheduling/Jobs/ScanFolderJob.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Quartz;
+
+namespace Shoko.Server.Scheduling.Jobs;
+
+public class ScanFolderJob : IJob
+{
+
+    public static readonly JobKey Key = new("ScanFolder", "System");
+    
+    public async Task Execute(IJobExecutionContext context)
+    {
+        var folderId = context.MergedJobDataMap.GetInt("importFolderID");
```

Comment:
See here: https://github.com/da3dsoul/quartznet/blob/f1e586f166d7588a06bf1ba7d545e6f8e98ffd8c/src/Quartz.Examples.AspNetCore/ExampleJob.cs

Quartz will automatically fill job data if they are properties on the IJob class.